app/cdap/components/ParsingDrawer/Components/ParsingHeaderActionTemplate/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Box } from '@material-ui/core';
+import { IMPORT_SCHEMA } from 'components/ParsingDrawer/constants';
+import { useStyles } from 'components/ParsingDrawer/styles';
+import React from 'react';
+
+const ParsingHeaderActionTemplate = (props) => {
```

Comment:
export default function and remove component name

Reply:
done